### PR TITLE
General maintenance

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Added
- Added MWL projects banner

### Changed
- Updated `league/route` and `league/container` (and other dependencies) to their latest stable versions
- Use now stable 1.0 of Composer dependency
- Update app to PSR7
- Use Webpack to bundle application (and remove Bower)

### Fixed
- Fixed link to Changehub page

### CODE REVIEW

File: app/Semver/Services/Packagist/Packagist.php [hunk 1]
```diff
@@ -28,7 +29,7 @@ class Packagist
     private $versionsRepository;
 
     /**
-     * @param VersionParser $parser
+     * @param VersionParser             $parser
```
Comment (hannesvdvreken) [inline app/Semver/Services/Packagist/Packagist.php]: Can we disable this alligning?

File: composer.json [hunk 2]
```diff
@@ -1,41 +1,49 @@
...
+		{
+			"name": "Maxime Fabre",
+			"email": "ehtnam6@gmail.com"
+		}
+	],
+	"require": {
+		"composer/composer": "^1.0",
+		"filp/whoops": "^2.1",
+		"illuminate/cache": "^5.2",
+		"illuminate/filesystem": "^5.2",
```
Comment (hannesvdvreken) [inline composer.json]: :+1: great upgrade of all dependencies! Next PR: replacing illuminate/cache with PSR-6

**Comment by hannesvdvreken** [comment]
Big PR to add one script tag :stuck-out-tongue:

:ship:

**Comment by Anahkiasen** [comment]
Yeah the project had been kind of left aside for a while so I figured I'd use the occasion to refresh it a bit while I was at it

File: composer.json (hunk 2 above)
Comment (Anahkiasen) [inline composer.json]: Is there already any usable implementation of it? Haven't seen any

File: app/Semver/Services/Packagist/Packagist.php (hunk 1 above)
Comment (Anahkiasen) [inline app/Semver/Services/Packagist/Packagist.php]: Kind of, I can disable it in PHPStorm but there is no way to enforce it project-wide through PCS

File: composer.json (hunk 2 above)
Comment (hannesvdvreken) [inline composer.json]: This is a pretty good one I think. Has been downloaded a lot too. https://packagist.org/packages/tedivm/stash

**Pull request merged**